src/components/measureList/MeasureList.tsx

```diff
@@ -312,6 +314,10 @@ export default function MeasureList(props: {
               errorData?.message ? errorData.message : "Server error!"
             );
           }
+          const message = JSON.parse(errorData?.request?.responseText)?.message;
+          if (message) {
+            setVersionHelperText(versionErrorHelper(message));
```

Comment:
This is just nice to have -> if we can add few screenshots of how the new feature looks on screen. Here it is hard to imagine what would be the value of 

error.response.request.responseText

Reply:
I see the error messages in utils. Cool